scripts/install-worker.sh

```diff
@@ -338,7 +416,7 @@ sudo yum install -y amazon-ssm-agent
 ### AMI Metadata ###############################################################
 ################################################################################
 
-BASE_AMI_ID=$(curl -s  http://169.254.169.254/latest/meta-data/ami-id)
+BASE_AMI_ID=$(get_meta_data 'latest/meta-data/ami-id')
```

Comment:
I didn't realize this was the only place IMDS is used during build. We could pass in the source AMI ID from packer and remove the IMDS dependency here.

tangent, but I'm also wondering if `/etc/eks/release` is actually useful to anyone. We tell folks to `cat /etc/eks/release` in our PR template, but we already know what kernel version, etc. shipped in a given release. Including the AL2 base AMI ID in that file seems more confusing than helpful

Reply:
I'm happy to remove  `/etc/eks/release` if it's not useful. In the caching PR, I'll be using metadata again so we'll still need these imds helper. I could move them to a `/usr/bin` was trying to keep everything self contained, but not opposed to sharing the helpers.

Reply:
@bwagner5 I threw something together here: https://github.com/awslabs/amazon-eks-ami/pull/1044

lmk your thoughts

Reply:
going to close in favor of #1044